tests/test_perf.py

```diff
@@ -151,3 +160,18 @@ def test_perf_thread_comm_is_process_comm(
         #   pative 1925947 [010] 987095.272656: PERF_RECORD_COMM: pative:1925904/1925947
         # we take the exec comm for all threads so we remain with the first, "native".
         _assert_comm_in_profile(profiler, application_pid, True)
+
+@pytest.mark.parametrize("runtime", ["native_thread_comm"])
+@pytest.mark.parametrize("perf_mode", ["fp"])
+@pytest.mark.parametrize("insert_dso_name", [False, True])
+@pytest.mark.parametrize("in_container", [True])
+def test_dso_name_in_perf_profile(
+    system_profiler: SystemProfiler,
+    application_pid: int,
+    insert_dso_name: bool,
+) -> None:
+    with system_profiler as profiler:
+        collapsed = snapshot_pid_profile(profiler, application_pid).stacks
+        logging.warning(f"{'HONYA' if insert_dso_name else 'NIKUYA'}: {pformat(collapsed, indent=2)}")
```

Comment:
Haha. Not sure I know what's the reference though. Can remove

Reply:
👍